test/diageig.jl

```diff
@@ -0,0 +1,31 @@
+using LinalgBackwards
+using Flux
+using Flux.Tracker: @grad, data, track, TrackedTuple
+using Test
+
+@testset "diageigbp" begin
```

Comment:
use the `Tracker.gradcheck` function in `Flux.jl` to check gradients. see other files under `test/` as examples.